### ISSUE TITLE
::MutableFile injects useless context for _added_by

### ISSUE THREAD

**Issue by kentfredric** [rjbs/Dist-Zilla#284]
In doing some work in reporting file changes, I discovered this:

```perl
Class::MOP::Method::Wrapped line 163
```

In the `_added_by` field for a file.

I was doing some weird things, so I did some isolated modifications to make sure I wasn't imagining things:

```perl
has added_by => (
  is => 'ro',
  writer => '_rset_added_by',   # was _set_added_by
  isa => 'Str',
);

# Hand written method added
sub _set_added_by {
    my ( $self, $value ) = @_;
    warn "Addedby $value";
    $self->_rset_added_by($value);
}
```

Output:

```
-[SNIP]-
[@Author::KENTNL/GithubMeta] skipping meta.resources.repository creation                                                                                                                                                                                        
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.                                                                                                               
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.                                                                                                               
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.                                                                                                               
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.     
-[SNIP]-
```

Yuck 

```perl
sub _set_added_by {
    my ( $self, $value ) = @_;
    Carp::cluck "Addedby $value" if $value =~ /Class::MOP/;
    $self->_rset_added_by($value);
}
```

```
[Bootstrap::lib] Bootstrapping lib
[DZ] beginning to build Dist-Zilla-Plugin-MetaData-FileModifiers
[DZ] guessing dist's main_module is lib/Dist/Zilla/Plugin/MetaData/FileModifiers.pm
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.
        Dist::Zilla::Role::File::_set_added_by(CODE(0x477b420), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::_set_added_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 106
        Dist::Zilla::Role::MutableFile::_update_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "encoded_content", "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 91
        Dist::Zilla::Role::MutableFile::encoded_content(CODE(0x44ab7c8), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::encoded_content(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/PPI.pm line 65
        Dist::Zilla::Role::PPI::save_ppi_document_to_file(Dist::Zilla::Plugin::PkgVersion=HASH(0x5d8f388), PPI::Document=HASH(0x3dac1f8), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PkgVersion.pm line 211
        Dist::Zilla::Plugin::PkgVersion::munge_perl(Dist::Zilla::Plugin::PkgVersion=HASH(0x5d8f388), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PkgVersion.pm line 89
        Dist::Zilla::Plugin::PkgVersion::munge_file(Dist::Zilla::Plugin::PkgVersion=HASH(0x5d8f388), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PkgVersion.pm line 78
        Dist::Zilla::Plugin::PkgVersion::munge_files(Dist::Zilla::Plugin::PkgVersion=HASH(0x5d8f388)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 324
        Dist::Zilla::Dist::Builder::build_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50), undef) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 390
        Dist::Zilla::Dist::Builder::ensure_built_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 379
        Dist::Zilla::Dist::Builder::ensure_built(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 443
        Dist::Zilla::Dist::Builder::build_archive(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/App/Command/build.pm line 66
        Dist::Zilla::App::Command::build::execute(Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740), ARRAY(0x242a1a8)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 282
        App::Cmd::execute_command(Dist::Zilla::App=HASH(0x2063780), Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 210
        App::Cmd::run("Dist::Zilla::App") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/bin/dzil line 15
[DZ] extracting distribution abstract from lib/Dist/Zilla/Plugin/MetaData/FileModifiers.pm
[@Author::KENTNL/GithubMeta] A remote named 'origin' was specified, but does not appear to exist.
[@Author::KENTNL/GithubMeta] skipping meta.resources.repository creation
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.
        Dist::Zilla::Role::File::_set_added_by(CODE(0x477b420), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::_set_added_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 106
        Dist::Zilla::Role::MutableFile::_update_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "content", "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 59
        Dist::Zilla::Role::MutableFile::content(CODE(0x44e0f68), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::content(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PodWeaver.pm line 141
        Dist::Zilla::Plugin::PodWeaver::munge_pod(Dist::Zilla::Plugin::PodWeaver=HASH(0x6011758), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PodWeaver.pm line 106
        Dist::Zilla::Plugin::PodWeaver::munge_file(Dist::Zilla::Plugin::PodWeaver=HASH(0x6011758), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/PodWeaver.pm line 98
        Dist::Zilla::Plugin::PodWeaver::munge_files(Dist::Zilla::Plugin::PodWeaver=HASH(0x6011758)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 324
        Dist::Zilla::Dist::Builder::build_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50), undef) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 390
        Dist::Zilla::Dist::Builder::ensure_built_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 379
        Dist::Zilla::Dist::Builder::ensure_built(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 443
        Dist::Zilla::Dist::Builder::build_archive(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/App/Command/build.pm line 66
        Dist::Zilla::App::Command::build::execute(Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740), ARRAY(0x242a1a8)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 282
        App::Cmd::execute_command(Dist::Zilla::App=HASH(0x2063780), Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 210
        App::Cmd::run("Dist::Zilla::App") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/bin/dzil line 15
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.
        Dist::Zilla::Role::File::_set_added_by(CODE(0x477b420), Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::_set_added_by(Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 106
        Dist::Zilla::Role::MutableFile::_update_by(Dist::Zilla::File::OnDisk=HASH(0x63fd150), "content", "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 59
        Dist::Zilla::Role::MutableFile::content(CODE(0x44e0f68), Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Release history for Dist-Zilla-Plugin-MetaData-FileModifiers\x{a}"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Release history for Dist-Zilla-Plugin-MetaData-FileModifiers\x{a}"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::content(Dist::Zilla::File::OnDisk=HASH(0x63fd150), "Release history for Dist-Zilla-Plugin-MetaData-FileModifiers\x{a}"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/NextRelease.pm line 113
        Dist::Zilla::Plugin::NextRelease::munge_files(Dist::Zilla::Plugin::Git::NextRelease=HASH(0x649b988)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 324
        Dist::Zilla::Dist::Builder::build_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50), undef) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 390
        Dist::Zilla::Dist::Builder::ensure_built_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 379
        Dist::Zilla::Dist::Builder::ensure_built(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 443
        Dist::Zilla::Dist::Builder::build_archive(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/App/Command/build.pm line 66
        Dist::Zilla::App::Command::build::execute(Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740), ARRAY(0x242a1a8)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 282
        App::Cmd::execute_command(Dist::Zilla::App=HASH(0x2063780), Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 210
        App::Cmd::run("Dist::Zilla::App") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/bin/dzil line 15
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.
        Dist::Zilla::Role::File::_set_added_by(CODE(0x477b420), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::_set_added_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 106
        Dist::Zilla::Role::MutableFile::_update_by(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "encoded_content", "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 91
        Dist::Zilla::Role::MutableFile::encoded_content(CODE(0x44ab7c8), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::OnDisk::encoded_content(Dist::Zilla::File::OnDisk=HASH(0x7b1aa28), "use 5.008; #utf8\x{a}use strict;\x{a}use warnings;\x{a}use utf8;\x{a}\x{a}package"...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/PPI.pm line 65
        Dist::Zilla::Role::PPI::save_ppi_document_to_file(Dist::Zilla::Plugin::Authority=HASH(0x6b8ac08), PPI::Document=HASH(0x8904560), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/Authority.pm line 192
        Dist::Zilla::Plugin::Authority::_munge_perl(Dist::Zilla::Plugin::Authority=HASH(0x6b8ac08), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/Authority.pm line 123
        Dist::Zilla::Plugin::Authority::_munge_file(Dist::Zilla::Plugin::Authority=HASH(0x6b8ac08), Dist::Zilla::File::OnDisk=HASH(0x7b1aa28)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/Authority.pm line 117
        Dist::Zilla::Plugin::Authority::munge_files(Dist::Zilla::Plugin::Authority=HASH(0x6b8ac08)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 324
        Dist::Zilla::Dist::Builder::build_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50), undef) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 390
        Dist::Zilla::Dist::Builder::ensure_built_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 379
        Dist::Zilla::Dist::Builder::ensure_built(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 443
        Dist::Zilla::Dist::Builder::build_archive(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/App/Command/build.pm line 66
        Dist::Zilla::App::Command::build::execute(Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740), ARRAY(0x242a1a8)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 282
        App::Cmd::execute_command(Dist::Zilla::App=HASH(0x2063780), Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 210
        App::Cmd::run("Dist::Zilla::App") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/bin/dzil line 15
Addedby Class::MOP::Method::Wrapped line 163 at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/File.pm line 45.
        Dist::Zilla::Role::File::_set_added_by(CODE(0x477b420), Dist::Zilla::File::InMemory=HASH(0x7e02c80), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::InMemory=HASH(0x7e02c80), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::InMemory::_set_added_by(Dist::Zilla::File::InMemory=HASH(0x7e02c80), "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 106
        Dist::Zilla::Role::MutableFile::_update_by(Dist::Zilla::File::InMemory=HASH(0x7e02c80), "content", "Class::MOP::Method::Wrapped line 163") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Role/MutableFile.pm line 59
        Dist::Zilla::Role::MutableFile::content(CODE(0x44e0f68), Dist::Zilla::File::InMemory=HASH(0x7e02c80), "#!perl\x{a}\x{a}use strict;\x{a}use warnings;\x{a}\x{a}# This test was generated "...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 163
        Class::MOP::Method::Wrapped::__ANON__(Dist::Zilla::File::InMemory=HASH(0x7e02c80), "#!perl\x{a}\x{a}use strict;\x{a}use warnings;\x{a}\x{a}# This test was generated "...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/x86_64-linux/Class/MOP/Method/Wrapped.pm line 92
        Dist::Zilla::File::InMemory::content(Dist::Zilla::File::InMemory=HASH(0x7e02c80), "#!perl\x{a}\x{a}use strict;\x{a}use warnings;\x{a}\x{a}# This test was generated "...) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/Test/ReportPrereqs.pm line 58
        Dist::Zilla::Plugin::Test::ReportPrereqs::_munge_test(Dist::Zilla::Plugin::Test::ReportPrereqs=HASH(0x55c25f0), Dist::Zilla::File::InMemory=HASH(0x7e02c80)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Plugin/Test/ReportPrereqs.pm line 65
        Dist::Zilla::Plugin::Test::ReportPrereqs::setup_installer(Dist::Zilla::Plugin::Test::ReportPrereqs=HASH(0x55c25f0)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 333
        Dist::Zilla::Dist::Builder::build_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50), undef) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 390
        Dist::Zilla::Dist::Builder::ensure_built_in(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 379
        Dist::Zilla::Dist::Builder::ensure_built(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/Dist/Builder.pm line 443
        Dist::Zilla::Dist::Builder::build_archive(Dist::Zilla::Dist::Builder=HASH(0x4afab50)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/Dist/Zilla/App/Command/build.pm line 66
        Dist::Zilla::App::Command::build::execute(Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740), ARRAY(0x242a1a8)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 282
        App::Cmd::execute_command(Dist::Zilla::App=HASH(0x2063780), Dist::Zilla::App::Command::build=HASH(0x420e528), Getopt::Long::Descriptive::Opts::__OPT__::2=HASH(0x4218740)) called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/lib/site_perl/5.19.8/App/Cmd.pm line 210
        App::Cmd::run("Dist::Zilla::App") called at /home/kent/perl5/perlbrew/perls/perl-5.19.8/bin/dzil line 15
[DZ] writing Dist-Zilla-Plugin-MetaData-FileModifiers in Dist-Zilla-Plugin-MetaData-FileModifiers-0.001000
[@Author::KENTNL/ReadmeAnyFromPod] overriding README.mkdn in root
[DZ] building archive with Archive::Tar::Wrapper
[DZ] writing archive to Dist-Zilla-Plugin-MetaData-FileModifiers-0.001000.tar.gz
```

**Comment by rjbs** [comment]
This issue could use a failing test.

**Comment by karenetheridge** [comment]
This might be fixed now, via e5fa403de73b1185e60e260070f7fbfa0d860c45

**Comment by rjbs** [comment]
Closed unless we get a failing test showing this is still busted.
